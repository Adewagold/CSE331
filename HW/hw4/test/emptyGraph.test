# Tests that a graph can be created.

# Create a graph
CreateGraph empty_graph

# Print the nodes in the graph
ListNodes empty_graph