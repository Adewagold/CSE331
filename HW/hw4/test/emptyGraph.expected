# Tests that a graph can be created.

# Create a graph
created graph empty_graph

# Print the nodes in the graph
empty_graph contains: